commands/util/translate.js

```diff
@@ -0,0 +1,78 @@
+const { Command } = require('discord.js-commando');
+const request = require('request-promise');
+
+const { version } = require('../../package');
+
+module.exports = class TranslateCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'translate',
+			aliases: ['t'],
+			group: 'util',
+			memberName: 'translate',
+			description: 'Translates text.',
+			throttling: {
+				usages: 5,
+				duration: 60
+			},
+
+			args: [
+				{
+					key: 'query',
+					prompt: 'what text do you want to translate?\n',
+					type: 'string'
+				},
+				{
+					key: 'to',
+					prompt: 'what languages do you want to translate to?\n',
+					type: 'string'
+				},
+				{
+					key: 'from',
+					prompt: 'what languages do you want to translate from?\n',
+					type: 'string',
+					default: ''
+				}
+			]
+		});
+	}
+
+	async run(msg, args) {
```

Comment:
Make 3 variables for `arg.query`, `arg.to`, and `arg.from` also maybe think about switching `from` and `to`.

`Text, from, to` rather than `Text, to, from`.